ServiceStation/WebApp/Views/Owners/Edit.cshtml

```diff
@@ -0,0 +1,42 @@
+﻿@model BusinessLayer.Models.DTO.Owner
```

Comment:
Better to use ViewModels for that (not need to do that now).

But I see that your BLL Models that in folder /BusinessLayer/Models/ have incorrect namespaces (the DTO level useless here - remove in all BLL models and fix usages).